tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -73,6 +73,7 @@ internal enum IntegrationId
         UnvalidatedRedirect,
         TestPlatformAssemblyResolver,
         StackTraceLeak,
-        XpathInjection
+        XpathInjection,
+        SystemReflection,
```

Comment:
IMHO ReflectionInjection should be more correct, as done with the rest of the vuln integrations

Reply:
fixed b5b0003